### DISCUSSION TITLE
Yi 1080p Home 9FUSY21E2R - Does not load Firmware

### DISCUSSION THREAD

**Discussion by thecifu** [roleoroleo/yi-hack-MStar#504] [General]
Hello, my camera is the subject model with firmaware 4.5.0.0C_2019 When I load the firmware: https://github.com/roleoroleo/yi-hack-MStar/releases/download/0.4.9/y203c_0.4.9.tgz it looks like which does it correctly. But I get into the YI Home app and it's still responsive everything as usual.
I understand that if YI Home responds, it means that the Firmware has not been loaded correctly. No?

Is there any way to know what is happening?

Regard,

**Comment by roleoroleo** [comment]
You have to login into the web interface and disable the cloud if you want to disable tYi Home app.
Use your browser and open the web interface at http://IP_OF_YOUR_CAM:8080

**Reply by thecifu** [reply]
The problem is that I can't access via the web nor does it respond to a PING or TELNET.
I leave you screenshot of the ip from the Router. https://tmpsee.com/v/bywz3duf7p

**Reply by roleoroleo** [reply]
Is the image the reservation you made or the table of IPs assigned by the router?
In this 2nd case, it's very strange that the cam acquires an IP but it doesn't reply to a ping.

**Reply by thecifu** [reply]
It is the table assigned by the router.
I don't really understand anything either. From the Yi Home app I seamlessly access everything as usual.

**Reply by thecifu** [reply]
I have tried trying to access from the mobile and I get the same result.
I have also made sure that I am connected to the same 2.4Ghz SSID band and neither.
https://tmpsee.com/v/fym0u5tqnj

I don't understand anything. From the YI HOME application, is it normal for the same serial firmware to continue appearing?

**Reply by roleoroleo** [reply]
If the ip address is registered on your router the connection is ok.
And the name assigned to the host (yi-hack) means that the hack worked properly.
Check your network connections.
